Task3/Task3_Atanas_Kisyov/src/python/car_wash/main.py

```diff
@@ -0,0 +1,38 @@
+from db_connection.config import USER_CREDENTIALS
+from db_connection.connection_pools import ReadOnlyPool, ReadWritePool
+from utils.parse_arguments import add_arguments
+
+
+def main():
+    """Interact with user"""
+
```

Comment:
all operations with argument should be in main method.

Reply:
I am not sure I understand. Can you provide more information, please?

Reply:
Skip this comment